numojo/core/ndstrides.mojo

```diff
@@ -162,12 +162,6 @@ struct NDArrayStrides(Stringable):
                 " column major `F` are supported"
             )
 
-    fn __copy__(mut self, other: Self):
```

Comment:
This function is helpful if I want to make a deep copy of the NDStrides, since we do not have `__copyinit__`. So I propose to rename it as `copy` (remove dunders) and keep it.